JSDemos/Demos/Charts/DrillDown/description.md

```diff
@@ -1,13 +1,9 @@
-This demo shows a drill-down chart that visualizes data on two hierarchical views. The main view displays the population breakdown by continent. When you click the bar, a detailed view reveals the most populated countries on the selected continent.
+When the data source has several hierarchical levels, you may want to enable users to navigate between them. The movement from the higher to the lower hierarchical level is called *"drilling down"*. The movement backwards is called *"drilling up"*.
 
-## Bind to Data
+In this demo, when a user clicks a group of tiles in the TreeMap, the component drills down. To implement this technique, follow the steps below:
```

Comment:
You refer only to drill-down in this introductory sentence. Then you talk about drilling-up in one of the list items. I think it makes more sense to talk about both ways in the introduction.

And we don't have to talk about "a user" when we refer to the demo. Demos are for the readers of this text, so we can simply say "you".

```suggestion
In this demo, you can  click/tap a group of TreeMap tiles to drill down. Once you go down a level, navigation links appear at the top of the TreeMap. Use then to drill back up. To implement this technique, follow the steps below:
```